File: java/parser/src/main/java/gov/lbl/parser/domain/Declaration.java
package gov.lbl.parser.domain;

public class Declaration {
    private String identifier;
    private Array_subscripts array_subscripts;
    private Modification modification;

    public Declaration(String identifier, Array_subscripts array_subscripts, Modification modification) {
        this.identifier = identifier;
        this.array_subscripts = array_subscripts;
        this.modification = modification;
    }
}

File: jsParser/domain/Declaration.js
class Declaration {
  constructor (identifier, array_subscripts, modification) {
    identifier != null && identifier != '' ? this.identifier = identifier : ''
    array_subscripts != null ? this.array_subscripts = array_subscripts : ''
    modification != null ? this.modification = modification : ''
  }
}
exports.Declaration = Declaration
